internal/lifecycle/lifecycle.go

```diff
@@ -53,9 +66,11 @@ func NewLifecycle(sampleDir, serviceName, gcrURL string) Lifecycle {
 	dockerfileExists := err == nil
```

Comment:
Technically there can be other errors from os.Stat other than the file not existing. It's probably fine not to distinguish but the variable names are slightly misleading.

Reply:
Resolved, thank you.